source/mobile/deploy-mobile-apps-using-emm-provider.rst

```diff
@@ -0,0 +1,131 @@
+Deploying Mobile Apps Using an EMM Provider
+===========================================
+
+An EMM Provider builds `Enterprise Mobile Management software <https://en.wikipedia.org/wiki/Enterprise_mobility_management>`__ that helps enterprise teams manage secure mobile endpoints with managed app configuration. 
+
+You can use an EMM to: 
+
+- Enforce users to download a mobile app.
```

Comment:
Would this read better as: "Control which mobile app users download."?

Reply:
@esethna - which statement is closer to the user's reality when using EMM providers?

Reply:
it should be directly associated with MM, something like

```
Established the Mattermost pre-built or custom apps to be managed by your organization
```

or something along those lines